examples/structured_2d_dgsem/elixir_eulerpolytropic_wave.jl

```diff
@@ -0,0 +1,83 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the polytropic Euler equations
+
+gamma = 2.0   # Adiabatic monatomic gas in 2d.
+kappa = 1.0     # Scaling factor for the pressure.
+equations = PolytropicEulerEquations2D(gamma, kappa)
+
+# Linear pressure wave in the negative x-direction.
+function initial_condition_wave(x, t, equations::PolytropicEulerEquations2D)
+    rho = 1.0
+    v1 = 0.0
+    if x[1] > 0.0
+        rho = ((1.0 + 0.01 * sin(x[1] * 2 * pi)) / kappa)^(1 / gamma)
+        v1 = ((0.01 * sin((x[1] - 1 / 2) * 2 * pi)) / kappa)
```

Comment:
Again, `equations.kappa` and `equations.gamma`?

Reply:
Yep. Better style.